### ISSUE TITLE
Visibility Rules Missing

### ISSUE THREAD

**Issue by velkymx** [ProcessMaker/screen-builder#360]

**Comment by velkymx** [comment]
We're noticing this causing issues with the submit buttons

**Comment by chipit24** [comment]
@caleeli It appears this is the line that is causing the problem: https://github.com/ProcessMaker/screen-builder/pull/363/files#diff-f631b3a2a589629965342504efc692b7R471.
```js
const control = this.controls.find(control => control.component === this.inspection.component) || this.inspection;
```

You're comparing `control.component` to `this.inspection.component`, but there are different controls with the same component name, as is the case with Submit Button and Navigation Button (both use `FormButton`). The above line seems to return the config for the navigation button when it should return the config for the submit button.

**Comment by JCook21** [comment]
This appears to have been fixed by #376.
